scrape_repo_information.py

```diff
@@ -0,0 +1,107 @@
+import collections
+import json
+import os
+import time
+import urllib
+import urllib.request
+import xml.etree.ElementTree as xml
+import sys
+
+"""
+Script to scrape text from GitHub and into JSON format for all language pairs in Apertium.
+Stem counter by sushain
+One entry in the JSON file would be
+{"lg1": "afr", "lg2": "nld", "last_updated": "2018-07-21T23:55:22Z", "created": "2012-02-08T15:12:04Z", "direction": "<>", "repo": "trunk", "stems": 6269}
+"""
+
+if "GITHUB_CLIENT_ID" in os.environ:
+    CLIENT_ID = os.environ["GITHUB_CLIENT_ID"]
+else:
+    CLIENT_ID = ""
+    print("Set Github Client Id in ENV variable to 'GITHUB_CLIENT_ID' There doesn't seem to be a GitHub Client ID; Please set an enviornment variable of GITHUB_CLINT_ID to your client id. the scraping process may not be fast get one here https://github.com/settings/applications/new")
+
+if "GITHUB_SECRET_CLIENT_ID" in os.environ:
+    CLIENT_SECRET = os.environ["GITHUB_SECRET_CLIENT_ID"]
+else:
+    CLIENT_SECRET = ""
+    print("Set Github Client secret Id in ENV variable to 'GITHUB_SECRET_CLIENT_ID' There doesn't seem to be a GitHub Client Secret; Please set an enviornment variable of CLIENT_SECRET to your client secret id. the scraping process may not be fast get one here https://github.com/settings/applications/new")
+
+Pair = collections.namedtuple("Pair", "lg1 lg2 last_updated created direction repo stems")
+types = ["trunk", "incubator", "nursery", "staging"]
+
+def get_bidix_stem_count(uri):
+    try:
+        dix_content = urllib.request.urlopen(uri).read().decode()
+        tree = xml.fromstring(dix_content)
+    except:
+        return -1 # FIXME: error handling
+
+    return len(tree.findall("*[@id='main']/e//l"))
+
+def main():
+    pairs = []
+    params = urllib.parse.urlencode(dict({"client_id": CLIENT_ID, "client_secret": CLIENT_SECRET}))
+    for repo_name in types:
+        types_data_url = "https://api.github.com/repos/apertium/apertium-%s/contents?" % repo_name
+        types_data_data = urllib.request.urlopen(types_data_url + params)
+        types_data = json.loads(types_data_data.read())
+        for type_content in types_data:
+            if "apertium" in type_content["name"] and (type_content["name"].count("-") == 2):
+                direction = ""
+                lang_pair_name = type_content["name"]
+                #getting names
+                _, lg1, lg2 = lang_pair_name.split("-")
+                if lg1 != lg2:
```

Comment:
Instead of having everything under an `if` here, just use a `continue` with the negated condition.